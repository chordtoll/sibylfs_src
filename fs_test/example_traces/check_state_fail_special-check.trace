@type trace

open_close "file" [O_CREAT] 0o700
Tau
RV_none

open_close "file" [O_TRUNC]
Tau
RV_none
