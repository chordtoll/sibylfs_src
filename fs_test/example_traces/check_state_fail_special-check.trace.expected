# processing file 'example_traces/check_state_fail_special-check.trace' ...
        @type trace

     3: open_close "file" [O_RDONLY;O_CREAT] 0o700
     4: Tau
     5: RV_none

     7: open_close "file" [O_RDONLY;O_TRUNC]
     8: Tau
# 
# Fatal error:      8: Tau
#    special states found:
#      - OS_special (Undefined, "open: O_TRUNC, no O_RDWR or O_WRONLY, posix/open.md O_TRUNC:4")
#    no normal result states

# trace not accepted
# processing file 'example_traces/check_state_fail_special-check.trace' failed
